packages/react-cosmos/src/components/proxies/PropsProxy.jsx

```diff
@@ -10,11 +10,15 @@ export default class PropsProxy extends React.Component {
       fixture,
       onComponentRef,
     } = this.props;
+    let children;
+    if (Array.isArray(fixture.children)) {
+      children = [].slice.call(fixture.children);
+    }
 
     return React.createElement(component, {
       ...fixture,
```

Comment:
My problem is due to a lack of experience using `enzyme`. I tried to modify the fixture to the following:

```
const fixture = {
  foo: 'bar',
  children: [{
    c1: {
      baz: 42,
    },
  }],
};
```

But I get the following error:

Invariant Violation: Objects are not valid as a React child (found: object with keys {c1}). If you meant to render a collection of children, use an array instead or wrap the object using createFragment(object) from the React add-ons.

That's why I assumed I was doing something wrong in `createElement`, but it's most likely to do with the shallow rendering of `enzyme`.